server/routes/dashboard.ts

```diff
@@ -115,6 +123,23 @@ export default function dashboard(service: Services): Router {
       toDate = null
       errors.push({ href: '#toDate', text: `Enter a valid to date, for example ${todayAsShortDate}` })
     }
+    let prisonerId: string = null
+    let referenceNumber: string = null
+    if (searchID) {
+      // Test if search is for a prisoner ID and use if so
+      if (searchID.match(/^[a-zA-Z][0-9]{4}[a-zA-Z]{2}$/)) {
```

Comment:
i'd recommend trimming whitespace around `searchId` so that "A1401AE " still works